packages/dx-react-core/docs/guides/providing-actions.md

```diff
@@ -0,0 +1,17 @@
+# React Core - Providing Actions
+
+Plugin host component allows to share actions between plugins.
+
+## Action
+
+[Action](../reference/action.md) is a plugin primitive that defines a function that can be executed outside of the plugin containing it. It is an invisible component. It uses the `name` property to identify the function.
+
+The main application of Action is a possibility to connect a result value to markup. [TemplateConnector](../reference/template-connector.md) is a vizualization primitive that serves this need.
```

Comment:
Seems like a Getter and Action make one functionality. Because this sentence is a duplicate on this guide and on the Action guide.
Maybe we should write - `The main application of Action is a possibility to connect a function who can change a value to markup.`